docs/markdown/Python/python-integrations/awslambda-python.md

```diff
@@ -26,6 +26,15 @@ backend_packages.add = [
 
 This adds the new `python_awslambda` target, which you can confirm by running `pants help python_awslambda`
 
+> 🚧 Set `layout = "zip"` for Pants 2.17
+>
+> Pants 2.17 is transitioning to a new, better layout, but defaults to the old Lambdex layout for backwards compatibility (see [below](#migrating-from-pants-216-and-earlier) for more details). To silence the warnings and be ready for Pants 2.18, add the following to the end of your `pants.toml`
+>
+> ```toml pants.toml
+> [lambdex]
+> layout = "zip"
+> ```
+
```

Comment:
Should probably mention here that the function handler name has changed, as that will be a big gotcha.